packages/workbox-core/utils/logger.mjs

```diff
@@ -54,5 +59,6 @@ const groupEnd = () => {
     console.groupEnd();
   }
 };
-
+export {setLoggerLevel};
```

Comment:
Is there any effective difference when using two individual exports of single-property objects vs. combining this and `getLoggerLevel` into a two-property object?

This is not blocking if this is just a style preference. If there *is* a difference, it would be good to add a comment explaining what the difference is.

Reply:
Just preference.